src/Form/index.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Story } from '@storybook/react';
+import { Input } from 'antd';
+import React from 'react';
+
+import { Button } from '../index';
+
+import { Form, FormProps } from './index';
+
+export default {
+  title: 'Form',
+  argTypes: {
+    bordered: { control: 'boolean' },
+    loading: { control: 'boolean' },
+  },
+};
+export const Default: Story<FormProps<string>> = (args) => (
+  <Form
+    {...args}
+    name="basic"
+    initialValues={{ remember: true }}
+    onFinish={(values: any) => {
+      // eslint-disable-next-line no-console
+      console.log('Success:', values);
+    }}
+    onFinishFailed={(errorInfo: any) => {
+      // eslint-disable-next-line no-console
+      console.log('Failed:', errorInfo);
+    }}
+  >
+    <Form.Item
+      label="Username"
+      name="username"
+      rules={[{ required: true, message: 'Please input your username!' }]}
```

Comment:
```suggestion
      rules={[{ required: true, message: 'Please input your username.' }]}
```

Let's not shout at our users 😉